charts/incubator/amd-automatic-music-downloader/values.yaml

```
@@ -0,0 +1,29 @@
+image:
+  repository: tccr.io/truecharts/amd-automatic-music-downloader
+  pullPolicy: IfNotPresent
+  tag: latest@sha256:9d0cf8206966456e065a1ded124b28dd0fb6172c5e6ded082e10788cfcadb24f
+
+securityContext:
+  readOnlyRootFilesystem: false
+  runAsNonRoot: false
+
+podSecurityContext:
+  runAsUser: 0
+  runAsGroup: 0
+
+service:
+  main:
+    ports:
+      main:
+        port: 8688
+        targetPort: 8688
```

```suggestion
service:
  main:
    enabled: false
    ports:
      main:
        enabled: false

probes:
  liveness:
    enabled: false
  readiness:
    enabled: false
  startup:
    enabled: false
```
Since it has no service, you can disable it, also disable probes so it won't keep restarting it because of the 'no service'